anna-sala/challenges/game-of-life/src/gameoflife.js

```diff
@@ -0,0 +1,154 @@
+let neighbours = 0;
+let rows = 20;
+let colums = 40;
+let created = false;
+let currentGeneration = [];
+let nextGeneration = [];
+let body = document.body;
+let td;
+let tr;
+let tbl;
+
+setInterval(runGame, 1000);
+
+const gridCopy = (currentGeneration) => {
+  for (let i = 0; i < currentGeneration.length; i++) {
```

Comment:
For simple loops such as this, better apply for..in or for..of loops.

Reply:
ok thanksss